sys-apps/rng-tools/files/rng-tools-5-fix-noctty.patch

```diff
@@ -0,0 +1,15 @@
+https://bugs.gentoo.org/556456
+
```

Comment:
patch needs a description of what it's doing and why (and example).  please see: https://dev.gentoo.org/~vapier/clean-patches